http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/HttpServletWithSessionListenerIT.java

```diff
@@ -51,6 +57,30 @@ public void sessionSecured() {
         thenCheckActiveSessionsEqualTo(1);
     }
 
+    @Test
+    @DisplayName("undertow servlet test http2")
+    public void underTowHttp2Servert(VertxTestContext vertxTestContext) {
+        final String HTTP_2_VERSION = "HTTP_2";
```

Comment:
`HttpVersion.HTTP_2.toString()`? (I didn't check it out, but I suppose you should compare in `assertEquals` response that is basically the `HttpVersion.HTTP_2` with constant in Vert.x? no biggie though